pom.xml

```
@@ -8,10 +8,10 @@
 		<version>2.7.1</version>
 		<relativePath/> <!-- lookup parent from repository -->
 	</parent>
-	<groupId>com.example</groupId>
-	<artifactId>internProject</artifactId>
+	<groupId>com.oi.laborant</groupId>
+	<artifactId>internproject</artifactId>
```

I would prefer com.orioninnovation.tr.tech:laborant

```suggestion
	<groupId>com.orioninnovation.tr.tech</groupId>
	<artifactId>laborant</artifactId>
```